jobs/test_e2e_errand/templates/bin/run.erb

```diff
@@ -15,40 +16,63 @@ export PATH="/var/vcap/packages/java8/bin:$PATH"
 
 [ ! -e /home/vcap/.sincedb ] || rm /home/vcap/.sincedb
 
-
-echo '' ; echo '==> Purging existing test data...'
-
-curl -s -X DELETE '<%= p('test_e2e_errand.elasticsearch_hostport') %>/_all/sample-logs--repo-commits' ; echo ''
-
+export QUERY=$(cat <<EOF
+{ "query": {
+    "filtered" : {
+        "query" : {
+            "term" : { "@source.host":"sample-logs--repo-commits" }
+        },
+        "filter" : {
+          "bool" : {
+            "must" : {
+                "range" : {
+                    "@timestamp" : {
+                        "from" : "$(date --iso-8601=ns)",
```

Comment:
Just realized it's effectively the test's start date, but still, the `@timestamp` should be the value of the committer date as defined by the [filters](https://github.com/logsearch/logsearch-boshrelease/blob/develop/src/sample-logs/repo-commits/logstash-filters.conf#L21) so I'm still missing how something is working here.